### DISCUSSION TITLE
Vibe - cross platform whisper app

### DISCUSSION THREAD

**Discussion by thewh1teagle** [openai/whisper#2293] [Show and tell]
Hey everyone!
I built the app Vibe so everyone can use whisper offline on every computer and transcribe privately and fast.
Vibe supports Windows, Linux and macOS. It has many features that make it easy to transcribe.
It has useful features such as exporting as various formats (txt,srt,vtt,json and more). Batch transcription. and more.

You can download it from https://thewh1teagle.github.io/vibe

<img width=600 src="https://github.com/user-attachments/assets/60b9543e-fdd7-4616-970d-bd011ef96673">

**Comment by thos-grol** [comment]
Free, no bullshit, no trials etc. Nice.

**Reply by stephanmullerNL** [reply]
Looks nice, but it seems quite buggy at the moment

**Comment by commandlinetips** [comment]
great app thank you .

**Comment by Lorzari** [comment]
Very interesting.  Will Test.  Some QUESTIONS:  In what folder does Vibe save files that are temporary, cache, etc.?  Concern is about System Drive C getting dangerously full, as what is caused with Adobe software.  Can the Vibe temporary / cache files be saved in another drive folder that is not in System Drive C?

**Comment by Lorzari** [comment]
To increase percentage of transcription accuracy, including better transcriptions of overlapping dialog, while better defining separate people speaking :  Can dual tracks of a recording made with two mics be imported into Vibe for a transcription?  If yes, with what specific steps?  (Dual tracks such as:  a)  Stereo left and right tracks, and  b) Two mono tracks, of the same conversation with two people speaking.)

**Comment by scottfly189** [comment]
thank you!!😄
